package/Dockerfile

```diff
@@ -100,6 +103,8 @@ RUN curl ${HELM_URL_V3} | tar xvzf - --strip-components=1 -C /usr/bin && \
 RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     curl -sLf ${!CHANNELSERVER_URL} > /usr/bin/channelserver && \
     curl -sLf ${!K3S_URL} > /usr/bin/k3s && \
+    mkdir -p /var/lib/rancher/k3s/agent/images/ && \
+    curl -sLf ${!K3S_IMAGE_TAR} > /var/lib/rancher/k3s/agent/images/${CATTLE_K3S_VERSION}-images.tar && \
```

Comment:
Shouldn't the final path be `.../k3s-airgap-images-$ARCH.tar` ? Thats how docs have it, but idk if k3s just picks up whatever tar is in that dir.

Reply:
The name is not important, any tar file in the dir will get extracted.